lib/model/game_state.dart

```diff
@@ -50,6 +52,16 @@ class GameState with ChangeNotifier {
     randomiser.shuffle(deck);
   }
 
+  void toggleDiscardView({bool closeWindow = false}) {
+    if (closeWindow) {
+      viewMode = ViewMode.NORMAL;
+    } else {
+      viewMode = ViewMode.DISCARD;
+    }
```

Comment:
Wondering if, instead of the `ViewMode` enum, we can get away with literally just a boolean here. 

Currently `ViewMode.SCRYING` isn't used (it's passed to `CardPile`, but not used so needn't be), and nor is `ViewMode.NORMAL` except in tests. So perhaps this could become:

```
  void toggleDiscardView() {
    viewingDiscard = !viewingDiscard;
    notifyListeners();
  }
```

Reply:
Yeah, I did wonder about refactoring scrying to work with the view mode as well. I'd rather that than keep having more and more booleans added, but can change this if you prefer it.

Reply:
Yeah if we're going to keep `ViewMode` I'd rather we plug it in properly - at the moment it's a bit weird because `SCRYING` is a vestigial thing that's never set or used, which is pretty misleading. Seeing it in the `enum` would definitely make you think it was driving the scrying logic, which it isn't atm.

Maybe we apply the rule of three and hold off for now? Agree that it would be bad to have 5 booleans along the lines of `scrying`, `viewingDiscard`, `viewRemovedFromPlay` etc etc, but there are only 2 right now (and in the scrying case it's not even a boolean, we're able to reuse the `scrying` list - which is quite neat as we need it anyway).